packages/react/src/combobox/root/AriaCombobox.tsx

```diff
@@ -1056,8 +1057,8 @@ export function AriaCombobox<Value = any, Mode extends SelectionMode = 'none'>(
     activeIndex,
     selectedIndex,
     virtual: true,
-    loopFocus: true,
-    allowEscape: !autoHighlightMode,
+    loopFocus,
+    allowEscape: loopFocus,
```

Comment:
Why was this changed? How does escape gets effected whether the focus is looping or not?